example/test/pages/startPage.ts

```diff
@@ -0,0 +1,33 @@
+import {browser, by, element} from 'protractor';
+import {stepAnnotation} from '../../../lib'; // use 'scenarioo-js' instead of '../../lib' in real project
+
+export class StartPage {
+  firstListItem = element(by.css('li#item_one'));
+  secondListItem = element(by.css('li#item_two'));
+  selectedItem = element(by.id('selected'));
+
+  constructor() {
+    browser.get('/index.html');
+  }
+
+  @stepAnnotation()
+  clickFirstListItem() {
+    return this.firstListItem.click();
+  }
+
+  /**
+   * By default, the @stepAnnotation takes the Class Name followed by the method name as a description
+   * (e.g. 'StartPage: clickSecondListItem').
+   * This behaviour can be overwritten by providing a custom description as follows.
+   */
+  @stepAnnotation("Custom message for this step") // this will result in the description: 'Custom message for this step'
```

Comment:
Could we rename the annotation please to just "step" ? I think "stepAnnotation" is unnecessary long, I see that it is an annotation like that.

Reply:
Oh - I see "step" is allready used --- hmmm, how about "scStep" or "reportStep" - any better idea?